supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,76 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Technical best practices
+
+* Include links only when necessary.
+* Use hyperlinks and use them consistently.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Do not use JavaScript links. 
+* Do not use Flash links.
+* Do not use links in Frames or iframes.
+
+
+.Link text best practices
+
+* Contextually describe what the user will find at the target location so that they can decide if they want to leave their current location.
+* Use a concise sentence or sentence fragment as the link text.
+* Avoid bare URLs. Bare URLs are unhelpful because they have no information about the target location.
+* Avoid irrelevant link text.
+
+
+
+
 [[internal-links]]
 == Internal links
 
+Internal links are links within www.redhat.com.
+
+.Example AsciiDoc: Internal links
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+To link to another guide or to a user story within the same guide:
+----
+For more information about <topic>, see link:<link>[<link_text>] in _<title_name>_.
+----
+
+[[rh-kb-links]]
+=== Linking to Red Hat Knowledgebase articles
+
+* Ensure that your product team supports linking to Knowledgebase articles.
```

Comment:
Hi @dfitzmau I added this point based on your comment, wdyt?

Reply:
Thanks, @ndeevy . Looks good. Thanks for adding this point.